common/autoware_auto_perception_rviz_plugin/include/object_detection/object_polygon_display_base.hpp

```diff
@@ -138,12 +139,12 @@ class AUTOWARE_AUTO_PERCEPTION_RVIZ_PLUGIN_PUBLIC ObjectPolygonDisplayBase
   std::optional<Marker::SharedPtr> get_shape_marker_ptr(
     const autoware_auto_perception_msgs::msg::Shape & shape_msg,
     const geometry_msgs::msg::Point & centroid, const geometry_msgs::msg::Quaternion & orientation,
-    const ClassificationContainerT & labels) const
+    const ClassificationContainerT & labels, const double line_width = 0.03) const
```

Comment:
```suggestion
    const ClassificationContainerT & labels, const double & line_width) const
```

I'm not sure but it is natual for me.